modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/view/DiscoverAdapter.kt

```diff
@@ -421,13 +422,15 @@ internal class DiscoverAdapter(
                 adapter.loadCategories(categoriesFilter)
             }
         }
+        fun setMostPopularCategoriesId(ids: List<Int>) {
+            this.mostPopularCategoriesId = ids
+        }
         private fun getMostPopularCategories(categories: List<CategoryPill>): List<CategoryPill> {
-            // True Crime, Comedy, Culture, History, Fiction, Technology
-            val mostPopularCategoriesId = setOf(19, 3, 13, 18, 17, 15)
+            if (mostPopularCategoriesId.isNullOrEmpty()) return categories
 
             return categories
-                .filter { it.discoverCategory.id in mostPopularCategoriesId }
-                .sortedBy { mostPopularCategoriesId.indexOf(it.discoverCategory.id) }
+                .filter { it.discoverCategory.id in mostPopularCategoriesId!! }
+                .sortedBy { mostPopularCategoriesId!!.indexOf(it.discoverCategory.id) }
```

Comment:
Can you create a temporary `val` or use `orEmpty()` instead of using `!!`. This will be less prone to create bugs during refactoring, etc.

Reply:
Sure! I've push forced the changes